example/clone-space.js

```diff
@@ -193,6 +196,9 @@ function clone (sourceSpace, destinationSpace) {
   });
 }
 
+function tap$ (fn) {
```

Comment:
Yeah that can be removed
On 2 Sep 2015 12:26 pm, "Sascha Depold" <notifications@github.com> wrote:

> In example/clone-space.js
> <https://github.com/contentful/contentful-management.js/pull/29#discussion_r38517615>
> :
>
> > @@ -193,6 +196,9 @@ function clone (sourceSpace, destinationSpace) {
> >    });
> >  }
> >
> > +function tap$ (fn) {
>
> not needed I guess?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/contentful/contentful-management.js/pull/29/files#r38517615>
> .
>